lib/file-info-view.coffee

```diff
@@ -9,35 +9,43 @@ class FileInfoView extends HTMLElement
     @currentPath.classList.add('current-path')
     @appendChild(@currentPath)
 
-    @handleCopiedTooltip()
-
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
-    clickHandler = =>
-      text = @getActiveItemCopyText()
+    clickHandler = (event) =>
+      altClick = if process.platform is 'darwin' then event.metaKey else event.ctrlKey
```

Comment:
We could, but I'd rather not from a UX perspective -- control clicking on OS X is a lot less natural than Command clicking; this is the pattern that Facebook's `hyperclick` package uses, too!